--- contrail-federation-web-ssl	2013-11-21 12:21:27.683855993 +0000
+++ contrail-federation-web-ssl-new	2013-11-21 12:26:30.679855988 +0000
@@ -1,6 +1,6 @@
 <IfModule mod_ssl.c>
 <VirtualHost *:443>
-    ServerName contrail-federation-web.contrail.eu
+    ServerName contrail-federation-web
     Alias /static /var/lib/contrail/federation/federation-web/static
     
     WSGIScriptAlias / /usr/lib/contrail/federation-web/bin/django.wsgi
