--- web.xml	2013-10-21 14:51:50.000000000 +0000
+++ web.xml-new	2013-11-21 10:16:48.034708835 +0000
@@ -21,7 +21,7 @@
             <param-value>true</param-value>
         </init-param>
     </filter>
-
+<!-- 
     <filter>
         <description>Auditing</description>
         <filter-name>Auditing</filter-name>
@@ -35,6 +35,7 @@
         <filter-name>Auditing</filter-name>
         <url-pattern>/*</url-pattern>
     </filter-mapping>
+-->
 
     <servlet>
         <servlet-name>oauth-rest-api</servlet-name>
